classes.js

```diff
@@ -0,0 +1,51 @@
+// QUESTION 1:
+class Human {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+  ageOneYear() {
+    this.age++;
+  }
+  eating() {
+    return `Mmm, mmm, mmm, I'm love'n it.`
+  }
```

Comment:
We want to log the line to the console so that when you call upon the `eating` function later, it will log the line rather than just returning it.
```
console.log("Mmm, mmm, mmm, I'm love'n it.")
amy.eating() // This will automatically log the line without needing to be wrapped in console.log()
```
Please let me know if this makes sense or if I'm being unclear!